holopy/scattering/theory/mielens.py

```diff
@@ -128,3 +131,33 @@ def raw_fields(self, positions, scatterer, medium_wavevec, medium_index,
         field_xyz *= np.exp(1j * particle_kz) / incident_field_x
         return field_xyz
 
+    def _create_calculator(
+            self, particle_kz=None, index_ratio=None, size_parameter=None):
+        field_calculator = MieLensCalculator(
+            particle_kz=particle_kz,
+            index_ratio=index_ratio,
+            size_parameter=size_parameter,
+            lens_angle=self.lens_angle,
+            **self.calculator_accuracy_kwargs)
+        return field_calculator
+
+
+class AberratedMieLens(MieLens):
+    def __init__(self, spherical_aberration=0.0, lens_angle=1.0,
+                 calculator_accuracy_kwargs={}):
+        super(AberratedMieLens, self).__init__()
+        self.lens_angle = lens_angle
+        self.spherical_aberration = spherical_aberration
+        self.calculator_accuracy_kwargs = calculator_accuracy_kwargs
```

Comment:
Two of these lines (`lens_angle` and `calculator_accuracy_kwargs`) are sort of redundant with the call to `super.__init__()`, except that you don't pass them through to the `MieLens` init. Is there a specific reason you chose to do things this way?